docs/get-and-store-the-shop-origin.md

```diff
@@ -0,0 +1,29 @@
+# Authenticate with OAuth
+
+To keep your embedded apps secure, you are required to lock all communications to the shop origin. The shop origin is the hostname for the current shop, which consists of the shop name followed by `myshopify.com`. The shop origin for the current session is contained in the `host` URL query parameter that’s appended to your application URL when your app is loaded inside the Shopify admin.
+
+The `host` parameter is an encoded version of the shop origin that is required since App Bridge version 2.0. If you are using App Bridge version 1.0, please use `shop` instead.
```

Comment:
Let's remove the v1 stuff since we should be trying to get them to use v2
```suggestion
The `host` parameter is an encoded version of the shop origin that is required since App Bridge version 2.0.
```